tests/unit/ExtendTimeLimit.spec.ts

```diff
@@ -91,10 +91,6 @@ describe('Time Limit Extension component', () => {
   })
 
   it('get correct court order file number.', () => {
-    const wrapper = wrapperFactory()
-    const extendTimeLimit = wrapper.vm as any // wrapper.vm type is Vue
-
-    expect(extendTimeLimit.courtOrderNumberText).toEqual('')
-    wrapper.destroy()
+    expect(store.getCourtOrderNumberText).toEqual('')
```

Comment:
Hmm, I could be wrong, but my feeling is that since it's a store getter there's no need to mount any component.  Is there a benefit to mounting a component when testing store getters?

Reply:
It's possible that mounting a component sets some store state.